src/GeneralStateTestsFiller/stCallCodes/touchAndGoFiller.yml

```diff
@@ -0,0 +1,49 @@
+---
+touchAndGo:
+  # The empty account 0xe4971 is being called (with gas 0). The test checks if the touched
+  # empty account is removed in the end and enabling point of this feature.
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 1
+    currentGasLimit: 1000000
+    currentNumber: 1
+    currentTimestamp: 1000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  expect:
+  - indexes:
+      data: !!int -1
+    network:
+    - '>=Frontier'
+    result:
+      095e7baea6a6c7c4c2dfeb977efac326af552d87:
```

Comment:
Why? If it exist or not depends on the "network".

Reply:
because this is what you test, that the address is cleaned. if it is not mentioned in result of expect section then no checks are done during filling, only the stateRoot hash will be verified but it might happen that root hash contain the result we don't expect. 

you can have different expect sections for different networks